templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car?</h1>
```

Comment:
Car?

Reply:
Oh. I copied this file and forgot to change this :)

Reply:
![image](https://user-images.githubusercontent.com/25363499/200548828-82158b08-19c9-4fe5-be43-6778bde4933b.png)